build.sh

```diff
@@ -84,8 +85,8 @@ apt-get -q update
 apt-get -q -y --purge install ubuntu-standard casper lupin-casper \
   laptop-detect os-prober linux-generic
 
-#
-dpkg -i meilix-metapackage*.deb
+# TODO: Update melix-metapackage to work with Xenial
+#dpkg -i meilix-metapackage*.deb
```

Comment:
This will prevent the metapackages from being installed to the live CD. The metapackages are an integral part of the OS and it is needed very much badly for the distro, so please fix it by adding compatibility, it's easy to do so!